crates/hc/Cargo.toml

```diff
@@ -27,10 +27,10 @@ path = "src/lib.rs"
 anyhow = "1.0"
 clap = { version = "4.0", features = ["derive", "cargo"] }
 lazy_static = "1.4"
-holochain_cli_bundle = { path = "../hc_bundle", version = "^0.4.0-dev.0"}
```

Comment:
Great, so the `release-automation` messes with the formatting when it edits versions